pkg/guestagent/guestagent_linux.go

```diff
@@ -103,6 +107,7 @@ type agent struct {
 // setWorthCheckingIPTablesRoutine sets worthCheckingIPTables to be false
 // when no NETFILTER_CFG audit message was received for the iptablesIdle time.
 func (a *agent) setWorthCheckingIPTablesRoutine(auditClient *libaudit.AuditClient, iptablesIdle time.Duration) {
+	logrus.Info("setWorthCheckingIPTablesRoutine(): monitoring netfilter audit events")
```

Comment:
Perhaps:
```suggestion
	logrus.Debug"setWorthCheckingIPTablesRoutine(): monitoring netfilter audit events")
```

Same for other logs too

Reply:
I'm using INFO messages for the other monitors (system time, kubernetes services). Why hide this monitor?

I think we need cleaner logging, maybe using specific logger or fields for every monitor instead of adding a prefix for every log, but this can be done later.